build/pr-netfx-validation.yaml

```diff
@@ -49,22 +59,13 @@ jobs:
         displayName: 'If above is partially succeeded, then fail'
         condition: eq(variables['Agent.JobStatus'], 'SucceededWithIssues')
 
-  - template: azure-pipeline.template.yaml
-    parameters:
-      name: 'netfx_tests_windows'
-      displayName: '.NET Framework Unit Tests (Windows)'
-      vmImage: 'windows-2019'
-      scriptFileName: build.cmd
-      scriptArgs: RunTestsNetFx471 incremental
-      outputDirectory: 'TestResults'
-      artifactName: 'netfx_tests_windows-$(Build.BuildId)'
-
-  - template: azure-pipeline.template.yaml
-    parameters:
-      name: 'netfx_451_tests_windows'
-      displayName: '.NET Framework 451 Unit Tests (Windows)'
-      vmImage: 'vs2017-win2016'
-      scriptFileName: build.cmd
-      scriptArgs: RunTestsNetFx451 incremental
-      outputDirectory: 'TestResults'
-      artifactName: 'netfx_451_tests_windows-$(Build.BuildId)'
+  # - template: templates/azure-pipeline.template.yaml
```

Comment:
Based on https://github.com/maksimkim/SpanNetty/pull/14/files/f1bbb23f6e916be9bd6b7fd350ca5be2ba55901d I am under the impression that Windows 2019 is working, but we are leaving it out here or am I missing something?

Can we re-enable it?